server/app.ts

```diff
@@ -155,6 +156,21 @@ const rallyPointInitPromise = rallyPointService.initialize(
     app.use(koaConvert(koaWebpackHot(getWebpackCompiler())))
   }
 
+  log.info('Testing connection to redis.')
```

Comment:
I would personally remove this line. I don't think it's terribly important to let developers know this is about to happen, and it might confuse them since they'll only see the conclusion of this declaration in case of an error.

But then again, I don't think it hurts _too_ much to have more logging, so I'm willing to change my mind on this if others agree ^^.

Reply:
I just dropped it in because it'll hang there for a bit if the connection is bad. It was an after thought though, best case it'll just save someone from `ctrl-c`'ing before they get a clear error message if they get impatient

Reply:
Yeah, I'm fine with it being there